beacon-chain/slasher/chunks.go

```diff
@@ -0,0 +1,309 @@
+package slasher
+
+import (
+	"context"
+	"fmt"
+	"math"
+
+	"github.com/pkg/errors"
+	types "github.com/prysmaticlabs/eth2-types"
```

Comment:
```suggestion
	"github.com/prysmaticlabs/eth2-types"
```
No need for explicit aliasing here, `eth2-types` will be available as `types` in code.